src/iop/borders.c

```diff
@@ -974,10 +1071,13 @@ void gui_init(struct dt_iop_module_t *self)
   dt_iop_borders_gui_data_t *g = IOP_GUI_ALLOC(borders);
   dt_iop_borders_params_t *p = (dt_iop_borders_params_t *)self->default_params;
 
+  g->basis = dt_bauhaus_combobox_from_params(self, "basis");
```

Comment:
You need `N_("basis")` as this is the first time basis word is introduced and so we need to register it as needed for translation.

Reply:
This is the variable name. The $DESCRIPTION in params gives the slider label and this will be marked for translation by introspection.

Reply:
I will never get this right :) Note there is instances of `N_("")` in `dt_bauhaus_combobox_from_params` (in `src/iop/channelmixerrgb.c` for example).

Reply:
Yep. If there _isn't_ an explicit $DESCRIPTION, then the params field name is used as label by default. And that will then need to be marked with N_() somewhere...

It wouldn't be "wrong" to do it here, just not needed. :-)